semgrep/tests/e2e/snapshots/test_help/test_help_text/--help/help.txt

```diff
@@ -7,6 +7,7 @@ Options:
 
 Commands:
   ci       The recommended way to run semgrep in CI
+  install  Install specific semgrep dependencies (Experimental)
```

Comment:
Suggestion: remove from help so it's obvious this is experimental and users don't put it in scripts